node/node.go

```diff
@@ -459,6 +459,7 @@ func NewNode(config *cfg.Config,
 			&pex.PEXReactorConfig{
 				Seeds:    splitAndTrimEmpty(config.P2P.Seeds, ",", " "),
 				SeedMode: config.P2P.SeedMode,
+				SeedDisconnectWaitPeriod: pex.DefaultSeedDisconnectWaitPeriod,
```

Comment:
should probably be moved to config

Reply:
Do you want to open an issue/add a todo? Or is this sth. that should be decided and done here (in this PR)?

Reply:
In this PR

Reply:
Is there anything blocking us from adding it to the config? Sounds like a good idea to me.

Reply:
Okay, I think we should not do that because

a) DefaultSeedDisconnectWaitPeriod depends on a private const blocksToContributeToBecomeGoodPeer
b) and average time between blocks, which may change during the course of the network

That said, I think it should be a function, not a const